plugins/basic/nexus-repository-maven/src/main/java/org/sonatype/nexus/repository/maven/internal/MavenHashCodes.java

```diff
@@ -19,44 +19,35 @@
 import javax.annotation.Nullable;
 
 import org.sonatype.nexus.common.hash.HashAlgorithm;
-import org.sonatype.nexus.repository.view.Content;
-import org.sonatype.nexus.repository.view.Payload;
 
 import com.google.common.collect.ImmutableMap;
 import com.google.common.collect.Maps;
 import com.google.common.hash.HashCode;
-import org.joda.time.DateTime;
 
 /**
- * Maven content.
+ * Maven hash codes map, usable as attribute.
  *
  * @since 3.0
  */
-public class MavenContent
-    extends Content
+public class MavenHashCodes
```

Comment:
Ack, I agree. @adreghiciu removing wrapper and moving them to more general place.